libs/ngx-mime/src/lib/core/intl/help-intl.lt.ts

```diff
@@ -17,4 +17,5 @@ export class HelpIntlLt extends HelpIntl {
   line9 =
     '<strong>F</strong>: Pilno ekrano režimas (Uždaroma paspaudus <strong>F</strong> arba <strong>ESC</strong>.)';
   line10 = '<strong>R</strong>: Pasukti 90 laipsnių';
+  line11 = '<strong>T</strong>:  Rodyti/slėpti optiškai atpažįstamą tekstą (turiniui, kurį galima rodyti)';
```

Comment:
To get screen readers to pause after an sentence, we should have a period punctuation at the end of each sentence.
Parenthesis are not read by the screen readers. 
PS: I have only tested with a free [Chrome extension](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn)

```suggestion
  line11 = '<strong>T</strong>:  Rodyti/slėpti optiškai atpažįstamą tekstą. (turiniui, kurį galima rodyti).';
```